avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastSerdeCache.java

```diff
@@ -122,10 +150,24 @@ public FastSerdeCache(Executor executorService) {
     this.compileClassPath = Optional.empty();
   }
 
+  public FastSerdeCache(Executor executorService, int limit) {
+    this(executorService);
+    this.generatedFastSerDesLimit = limit;
+  }
+
   private FastSerdeCache() {
     this((Executor) null);
   }
 
+  /**
+   * @param limit
+   *            custom number {@link #generatedFastSerDesLimit}
+   */
+  public FastSerdeCache(int limit) {
```

Comment:
Do we need to add other two constructors? Maybe this one is good enough.

Reply:
No harm to add these two constructors :) For exmaple, I am using `FastSerdeCache(Executor executorService, int limit)` in the test.